### PR TITLE
Fix invalid client certificate issue

### PR DESCRIPTION
**Pull request by elgnay**
Signed-off-by: Yang Le <yangle@redhat.com>

### CODE REVIEW

File: pkg/clientcert/certificate.go [hunk 1]
```diff
@@ -80,16 +80,18 @@ func IsCertificateValid(certData []byte, subject *pkix.Name) (bool, error) {
...
 	// check subject of certificates
+	organizationSets := sets.NewString(subject.Organization...)
+	organizationalUnitSets := sets.NewString(subject.OrganizationalUnit...)
 	for _, cert := range certs {
 		if cert.Subject.CommonName != subject.CommonName {
 			continue
 		}
 
-		if !reflect.DeepEqual(cert.Subject.Organization, subject.Organization) {
+		if !organizationSets.Equal(sets.NewString(cert.Subject.Organization...)) {
```
Comment (qiujian16) [inline pkg/clientcert/certificate.go]: i would want removing the comparison of o/ou now. We can add it when we think it is necessary.

**Review by qiujian16 — COMMENTED** [review]
/approve

File: pkg/clientcert/certificate.go (hunk 1 above)
Comment (elgnay) [reply pkg/clientcert/certificate.go]: Removed.

**Comment by skeeey** [comment]
refer to https://github.com/open-cluster-management/backlog/issues/11314

**Pull request merged**